v3/src/data-model/attribute.ts

```diff
@@ -48,6 +48,7 @@ export const Attribute = types.model("Attribute", {
   clientKey: "",
   sourceID: types.maybe(types.string),
   name: types.string,
+  userType: types.maybe(types.enumeration(["numeric", "nominal"])),
```

Comment:
Note that the user sees categorical, numeric, date, qualitative, boundary and checkbox as allowable types. Let's not use 'nominal', only 'categorical', to eliminate ambiguity compared to V2. I would think we would want to include all possibilities in this enumeration.

Reply:
I used `nominal` for consistency with CODAP2 because of this comment from `attribute_model.js`:
```
      /**
       * Type of the attribute: 'numeric' or 'nominal'
       */
      type: null,
```
but a closer look at the code suggests that both `nominal` and `categorical` are used internally so I can change it.